### ISSUE TITLE
Steps to do a manual full uninstall

### ISSUE THREAD

**Issue by fititnt** [tsuru/tsuru#2316]
Related: https://github.com/EticaAI/aguia-pescadora/issues/6

---

I'm testing `tsuru version 1.7.0-rc1` and I the `tsuru install-remove` in my case seems to not work and tsuru try to update older cluster data by default.

My system had references even from 2015 (`~/.tsuru_token`, `~/.tsuru_target` & `~/.tsuru_targets`, was from hackinpoa	https://api.hackinpoa.tsuru.io), and also I'm reusing the same name of target for a different project (at first, just tried to rename the `~/.tsuru` folder). 

So, my question is:

1. **How did I uninstall (or rename) all data for the tsuru-client does not know old targets?** (also, is good to know if is need to open a new terminal or reboot the local system to be sure, or if just renaming files/folder is ok.)
2. **What to manually remove on the target servers to allow a fresh reinstall?**
   - Just implode docker data?

----

**Environment information**

* Operating System: Ubuntu 18.04
* Tsuru installation method: `tsuru install-create`
* Tsuru API version: 'tsurud: command not found'
* Tsuru client version:tsuru version 1.7.0-rc1.

**Cluster information**

* Node provider: unmanaged
* Machine type: VM
* Machine specs (CPU, memory, disk, etc): 
    * VPS Águia Pescadora Delta [6vCPU / 16GB RAM / 400GB] 
    * VPS Águia Pescadora Echo [4vCPU / 8GB RAM / 200GB]
    * VPS Águia Pescadora Foxtrot [4vCPU / 8GB RAM / 200GB]
* Provisioner (`default`/`kubernetes`/`swarm`): default I guess
* Docker version: 18.09.6
* Kubernetes: No Kubernetes

**Comment by guilhermebr** [comment]
@fititnt `tsuru install-remove` is used to remove tsuru server created by `tsuru install-create`.
You can remove using the config file used to create: `tsuru install-remove -c intall-config.yml`

you can remove targets with `tsuru target-remove`

**Comment by fititnt** [comment]
**TL;DR: For who uses `tsuru install-create -c config.yml` `tsuru install-remove -c config.yml`. This command will not unstall or erase data on tsuru nodes**

----

Hummm... makes sense! I was missing the `-c config.yml`, so even the local files did not get removed and the `The following core machines will be destroyed:` return no data.

Also, for who is interested in this, tsuru asks twice. Here my example to help everyone who is testing.

```bash
# fititnt at bravo in /alligo/code/eticaai/aguia-pescadora/diario-de-bordo/tsuru-inicializacao on git:master o [19:12:57]
$ tsuru install-remove -c config.yml
This will uninstall Tsuru installed on your target aguia-pescadora: http://173.249.10.99:8080.
The following core machines will be destroyed:
+-------------------+----------------+---------------------------------------------------------------+
| Name              | IP             | Data                                                          |
+-------------------+----------------+---------------------------------------------------------------+
| aguia-pescadora-1 | 173.249.10.99  | {                                                             |
|                   |                | "EnginePort": 2376,                                           |
|                   |                | "IPAddress": "173.249.10.99",                                 |
|                   |                | "MachineName": "aguia-pescadora-1",                           |
|                   |                | "SSHKey": "/home/fititnt/.ssh/id_rsa-aguia-pescadora-tsuru",  |
|                   |                | "SSHKeyPath": "/tmp/106053189/machines/id_rsa",               |
|                   |                | "SSHPort": 22,                                                |
|                   |                | "SSHUser": "root",                                            |
|                   |                | "StorePath": "/home/fititnt/.tsuru/installs/aguia-pescadora", |
|                   |                | "SwarmDiscovery": "",                                         |
|                   |                | "SwarmHost": "",                                              |
|                   |                | "SwarmMaster": false                                          |
|                   |                | }                                                             |
+-------------------+----------------+---------------------------------------------------------------+
| aguia-pescadora-2 | 167.86.127.220 | {                                                             |
|                   |                | "EnginePort": 2376,                                           |
|                   |                | "IPAddress": "167.86.127.220",                                |
|                   |                | "MachineName": "aguia-pescadora-2",                           |
|                   |                | "SSHKey": "/home/fititnt/.ssh/id_rsa-aguia-pescadora-tsuru",  |
|                   |                | "SSHKeyPath": "/tmp/106053189/machines/id_rsa",               |
|                   |                | "SSHPort": 22,                                                |
|                   |                | "SSHUser": "root",                                            |
|                   |                | "StorePath": "/home/fititnt/.tsuru/installs/aguia-pescadora", |
|                   |                | "SwarmDiscovery": "",                                         |
|                   |                | "SwarmHost": "",                                              |
|                   |                | "SwarmMaster": false                                          |
|                   |                | }                                                             |
+-------------------+----------------+---------------------------------------------------------------+
| aguia-pescadora-3 | 167.86.127.225 | {                                                             |
|                   |                | "EnginePort": 2376,                                           |
|                   |                | "IPAddress": "167.86.127.225",                                |
|                   |                | "MachineName": "aguia-pescadora-3",                           |
|                   |                | "SSHKey": "/home/fititnt/.ssh/id_rsa-aguia-pescadora-tsuru",  |
|                   |                | "SSHKeyPath": "/tmp/106053189/machines/id_rsa",               |
|                   |                | "SSHPort": 22,                                                |
|                   |                | "SSHUser": "root",                                            |
|                   |                | "StorePath": "/home/fititnt/.tsuru/installs/aguia-pescadora", |
|                   |                | "SwarmDiscovery": "",                                         |
|                   |                | "SwarmHost": "",                                              |
|                   |                | "SwarmMaster": false                                          |
|                   |                | }                                                             |
+-------------------+----------------+---------------------------------------------------------------+
Are you sure you sure you want to uninstall tsuru? (y/n) y
Are you really sure? I wont ask you again. (y/n) y
Core Machines successfully removed!
removing target
Uninstall finished successfully!
```


But just to cite as future reference, for good or for bad, I think this command only removes the local references that exist a remove target (as described https://tsuru-client.readthedocs.io/en/latest/reference.html#list-existing-targets), but the Tsuru will still installed with all data on all remote nodes.

Here a example that shows the remote with all data.

```sh
root@aguia-pescadora-1:~# docker ps
CONTAINER ID        IMAGE                                             COMMAND                  CREATED             STATUS              PORTS                     NAMES
0d82b5b7507c        173.249.10.99:5000/tsuru/app-tsuru-dashboard:v1   "/bin/sh -lc '[ -d /…"   16 hours ago        Up 16 hours         0.0.0.0:32768->8888/tcp   tsuru-dashboard-6aa42a2bd28473cd2878
de336d990986        tsuru/bs:v1                                       "/bin/bs"                16 hours ago        Up 16 hours                                   big-sibling
c4d21086db5b        tsuru/api:v1                                      "/bin/tsurud api"        16 hours ago        Up 16 hours         8080/tcp                  tsuru_tsuru.1.fycgfefw7scvlqysqab0wc6ef
a42cde417eac        mongo:latest                                      "docker-entrypoint.s…"   16 hours ago        Up 16 hours         27017/tcp                 tsuru_mongo.1.06ptlj2zzex5n6mxpxixeujki
root@aguia-pescadora-1:~# 
```

**Comment by fititnt** [comment]
**For anyone reading this issue later, this is not a bug**. I made an error when opening the issue. If have some change at best could be extra explanation on tsuru documentation.

A full manual uninstall (erasing even data on remote servers) could be not something that should be that easy to be be done by command line. Maybe give me some extra days and I eventually come here with tips for this in English.
